airflow/providers/google/leveldb/operators/leveldb.py

```diff
@@ -92,7 +92,7 @@ def execute(self, context) -> Optional[str]:
             keys=self.keys,
             values=self.values,
         )
-        self.log.info("Done. Returned value was: %s", str(value))
+        self.log.debug("Done. Returned value was: %s", str(value))
         leveldb_hook.close_conn()
         value = value if value is None else value.decode()
```

Comment:
Here I also think we need to do more: str(value) will actually perform string conversion no matter what is the log value (and it is not needed anyway because %s will do it anyway. So here both 'if' and removing str is needed.